modules/nextMillenniumBidAdapter.js

```diff
@@ -245,8 +248,10 @@ export const spec = {
 };
 
 export function getImp(bid, id) {
+  const bidfloorcur = getCurrency();
```

Comment:
I don't think floors currency and adserver currency have to be the same; can't uou get this from getFloor?